style.css

```diff
@@ -0,0 +1,49 @@
+.block1{
```

Comment:
`block1`, `block4`,  `block7` have similar styles, create one common class, the same with others